pkg/storage/internalstorage/collectionresource_storage.go

```diff
@@ -39,7 +39,10 @@ func (s *CollectionResourceStorage) Get(ctx context.Context, opts *pediainternal
 
 		types[rt.GroupResource()] = &cr.ResourceTypes[i]
 	}
-	query = applyListOptionsToQuery(query, opts)
+
+	// collection resources don't support attach remaining count
+	opts.AttachRemainingCount = nil
```

Comment:
It is not good to change the original data when using, whether to add a unified check